config/dist/config.py.dist

```diff
@@ -1,72 +0,0 @@
-import os
```

Comment:
this file (config/dist/config.py.dist) was deleted? Can you add this file back in?

Reply:
The file was put back in.